### PR TITLE
Finished textfield

### PR DESCRIPTION
**Pull request by Jkierem**

### CODE REVIEW

File: src/TextField/index.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
+import getClassName from 'getclassname';
+import React, { useState } from 'react';
+import { Icons } from '../sharedTypes';
+
+interface TextFieldPropsWithoutRef {
+  /**
+   * id attribute to be passed to the underlying input element. Will default to `name`, then `label` props
```
Comment (suabochica) [inline src/TextField/index.tsx]: Minor detail, try to be consistent with the comments case, b/c some are capitalized and others are in lowercase. 🙏🏾 This convention is also reflected in the `README.md` file

File: stories/shared/IconOptions.tsx [hunk 2]
```diff
@@ -54,4 +54,4 @@ export const Icons = [
 , "trash"
 , "share"
 , "mail"
-, "lock"]
+, "lock"].sort()
```
Comment (suabochica) [inline stories/shared/IconOptions.tsx:63]: We can add a comment saying that the default sort order is ascending.

**Review by suabochica — APPROVED** [review]
LGTM!

File: src/TextField/index.tsx (hunk 1 above)
Comment (Jkierem) [reply src/TextField/index.tsx]: Sure thing. Also I made it more consistent with formatting

File: stories/shared/IconOptions.tsx (hunk 2 above)
Comment (Jkierem) [reply stories/shared/IconOptions.tsx:63]: Added in next commit

**Pull request merged**